packages/ckeditor5-emoji/CONTRIBUTING.md

```diff
@@ -0,0 +1,4 @@
+Contributing
+========================================
```

Comment:
I know that this `=` line does not match the header in most other packages, but this is so ugly...

```suggestion
Contributing
============
```